sched/scheduler/stateful_scheduler.go

```diff
@@ -56,7 +56,7 @@ const DefaultMaxJobsPerRequestor = 100
 const DefaultSoftMaxSchedulableTasks = 1000
 
 // Increase the NodeScaleFactor by a percentage defined by 1 + (Priority * NodeScaleAdjustment)
-var NodeScaleAdjustment = 1.0
+var NodeScaleAdjustment = .75
```

Comment:
Curious - what are you taking into account when adjusting this value? Could you add a short comment?